docs/project_setup.rst

```diff
@@ -101,8 +101,7 @@ Adding Tequila to a Django Project
    catch-all playbook that uses the include directive to pull in at
    least the common, db, web, worker, and queue playbooks).  Other
    playbooks should be created as needed to fill other
-   project-specific needs.  **TODO:** Create this playbooks/ directory
-   and populate.
+   project-specific needs.
```

Comment:
Would it make sense to mention the directory of typical playbooks here, or perhaps somewhere else?

Reply:
Line 94.

Reply:
👀